src/forklift/core.py

```diff
@@ -259,7 +239,18 @@ def _hash(crate, hash_path):
     attribute_hashes, geometry_hashes = _get_hash_lookups(crate.name, hash_gdb_path)
     unique_salt = 0
 
-    with arcpy.da.SearchCursor(crate.source, fields, sql_clause=sql_clause) as cursor:
+    #: TODO is there a way to not create the temp table and nested insert cursor if it doesn't need reprojecting and stay DRY
```

Comment:
I think we can do it if we don't use `with` and instead get a reference to the insert cursor if we need to create it.